lib/game.rb

```diff
@@ -0,0 +1,76 @@
+# frozen_string_literal: true
+
+require_relative 'frame'
+
+class Game
+  attr_reader :frames, :scores
+
+  def initialize
+    @frames = []
+    @scores = {}
+  end
+
+  def new_frame
+    frames << Frame.new(frames.length + 1)
+    scores["frame#{@frames.length}".to_sym] = 0
+  end
+
+  def latest_roll(pins)
+    frame_check
+    frames[-1].add_roll(pins)
+    scores["frame#{@frames.length}".to_sym] += pins
+    post_roll_score
```

Comment:
This won't properly calculate the score if you roll 2  or more strikes . In the irb it can only calculate a gam without spares or strikes as a running total